packages/pigeon/example/app/ios/Runner/AppDelegate.swift

```diff
@@ -37,16 +37,41 @@ private class PigeonFlutterApi {
   }
 
   func callFlutterMethod(
-    aString aStringArg: String?, completion: @escaping (Result<String, Error>) -> Void
+    aString aStringArg: String?, completion: @escaping (Result<String, PigeonError>) -> Void
   ) {
     flutterAPI.flutterMethod(aString: aStringArg) {
-      completion(.success($0))
+      completion($0)
     }
   }
 }
 // #enddocregion swift-class-flutter
 
-@UIApplicationMain
+// #docregion swift-class-event
+class SendEvents: StreamEventsStreamHandler {
+  var timerActive = false
+
+  override func onListen(withArguments arguments: Any?, sink: PigeonEventSink<SealedBaseClass>) {
```

Comment:
"Sealed" name coming from Kotlin? I've never seen this usage in Swift

Reply:
It's coming from dart, it's semantically meaningless here, but it has to have the same name in all locations.